tests/test_codemod_docs.py

```diff
@@ -12,10 +12,11 @@ def pytest_generate_tests(metafunc):
 
 
 def test_load_codemod_docs_info(codemod: BaseCodemod):
-    if codemod.name in ["order-imports", "refactor-new-api"]:
+    print(codemod.name)
```

Comment:
Leftover print statement.

Reply:
I should make a codemod for that and make it a pre-commit hook.